src/components/EventsList.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react';
+import EventLink from './EventLink';
+import styles from '../assets/CSS/sharedStyles.css';
+
+
+class EventsList extends Component {
+   // constructor(props){
+   //    super(props);
+   //    this.state = {
+
+   //    }
+   // }
+   render(){
+      return(
+         <ul >
+          <div 
+          // onClick={() => this.props.navigation.navigate('EventDetail', {
+          //  name: 'Volleyball',
+          //  location: 'The Quad',
+          //  date: '2/23/19',
+          //  startTime: '1:45PM', 
+          //  endTime: '3:15PM',
+          //  description: 'Something super cool',
+          //  category: 'Sports', 
+          //  creator: 'jpauga',  
+          //  spotsTaken: '6', 
+          //  spotsAvailable: '10', 
+          //  })}
```

Comment:
If we're not going to use this then let's delete it. If we are planning on implementing it in the future then we can leave it here